### ISSUE TITLE
pytorch VS tensorflow

### ISSUE THREAD

**Issue by vmtmxmf5** [vmtmxmf5/Pytorch-#1]
## Pytorch 명령어

| tf | torch |
|--|--|
| tf.Variable() | torch.FloatTensor() <br> torch.LongTensor() |
| axis | dim |
| a.shape | a.shape <br> a.size() |
| a.reshape() | a.view() <br> a.reshape() |
| a[tf.newaxis, ...] | a[None, ...] <br> a.unsqueeze(dim) | 
| tf.concat([x, y], axis=) | torch.cat([x, y], dim=) |
| tf.zeros_like() | torch.zeros_like() |

### 덮어쓰기 기능

a.mul_(2)

### index_fill

tmp.index_fill_(dim, index=torch.tensor([0,2]), val)

### 병합

```ruby
1. torch.cat할 때 dim=0의 사이즈만 자유롭고 나머지는 같아야 함
```

```ruby
2. cat vs stack
```
```xonsh
torch.cat([a.unsqueeze(0), b.unsqueeze(0), c.unsqueeze(0)])
torch.stack([a,b,c])
```

### random uniform dist.
torch.empty(4, 2).uniform_(0,1)
